compoments/report-repair/repair-availability.js

```diff
@@ -8,13 +8,12 @@ import moment from 'moment';
 const RepairAvailability = ({handleChange, values, nextAvailability}) => {
   const [error, setError] = useState();
   const [value, setValue] = useState();
-  console.log(values)
   const baseURL = '/api/availability';
   const params =  {
     repairLocation:  values.repairLocation,
     repairProblem:  values.repairProblem,
     repairIssue: values.repairProblemBestDescription,
-    locationId: values.address,
+    locationId: values.address?.locationId,
```

Comment:
Could `values.address` be null?
If so, given `locationId` is a required parameter (as are the others), then we should avoid calling the API when any required parameter is not set.

Reply:
this was more to the page could render without data for testing and local dev